pkg/network/ovn_kubernetes_test.go

```diff
@@ -262,6 +262,34 @@ func TestOVNKubernetesIsSafe(t *testing.T) {
 	errs := isOVNKubernetesChangeSafe(prev, next)
 	g.Expect(errs).To(BeEmpty())
 
+	// try to add a new hybrid overlay config
+	hybridOverlayConfigNext :=
+		operv1.HybridOverlayConfig{
+			HybridClusterNetwork: []operv1.ClusterNetworkEntry{
+				operv1.ClusterNetworkEntry{CIDR: "00.0.0.0", HostPrefix: 80},
+			},
+		}
+	next.DefaultNetwork.OVNKubernetesConfig.HybridOverlayConfig = &hybridOverlayConfigNext
+
+	errs = isOVNKubernetesChangeSafe(prev, next)
+	g.Expect(errs).To(HaveLen(1))
+	g.Expect(errs[0]).To(MatchError("cannot start a hybrid overlay network after install time"))
+
+	//try to change a previous hybrid overlay
+	hybridOverlayConfigPrev :=
+		operv1.HybridOverlayConfig{
+			HybridClusterNetwork: []operv1.ClusterNetworkEntry{
+				operv1.ClusterNetworkEntry{CIDR: "00.0.0.1", HostPrefix: 80},
+			},
+		}
+	prev.DefaultNetwork.OVNKubernetesConfig.HybridOverlayConfig = &hybridOverlayConfigPrev
+	errs = isOVNKubernetesChangeSafe(prev, next)
+	g.Expect(errs).To(HaveLen(1))
+	g.Expect(errs[0]).To(MatchError("once set cannot change ovn-kubernetes Hybrid Overlay Config"))
```

Comment:
Once set, cannot....
Maybe change string in the tested code?

Reply:
changed to "cannot edit a running hybrid overlay network"